scala-conductr-client-lib/src/main/scala/com/typesafe/conductr/clientlib/scala/models/BundleAttributes.scala

```diff
@@ -13,6 +13,7 @@ final case class BundleAttributes(
   roles: SortedSet[String],
   bundleName: String,
   systemVersion: String,
-  compatibilityVersion: String
+  compatibilityVersion: String,
+  tags: Option[Seq[String]]
```

Comment:
Tags aren't really optional... they're always there as of 2.1. This should be just a `Seq[String]` as per ConductR core.